unst_msh_gen/README.md

```diff
@@ -0,0 +1,42 @@
+# Project Name
+Unstructured mesh generation for WW3 using JIGSSAW.
+## Description
+Mesh generation script capable of creating unstructured meshes for WW3 global modeling. The tool leverages JIGSAWPY (https://github.com/dengwirda/jigsaw-python) for efficient triangulation.
+Main changes include:
+Implementation of the ocn_ww3.py to create uniform unstructured mesh for global WW3 model.
+This tool is under active development, with future work focused on variable unstructured mesh generation.
+
+## Installation
+
+1- Install jigsawpy (https://github.com/dengwirda/jigsaw-python)
+2- you need following packages:
+	numpy
+        scipy
+	packaging
+	netcdf4
+3- get the DEM:
+	$wget https://github.com/dengwirda/dem/releases/download/v0.1.1/RTopo_2_0_4_GEBCO_v2023_60sec_pixel.zip
+4- unzip the DEM 
+	unzip *.zip
+ 
+
+
+## Usage
+1- clone the repo
+	$mkdir meshgen
+	$cd meshgen
+	$git clone https://github.com/AliS-Noaa/WW3-tools/tree/add-unst-mesh-gen
+2- get the DEM:
+        $wget https://github.com/dengwirda/dem/releases/download/v0.1.1/RTopo_2_0_4_GEBCO_v2023_60sec_pixel.zip
+        $unzip *.zip
+3- run the script:
+	$python3 ocn_ww3.p --black_sea [option]
+		
+		option=  3: default which will have the Black Sea and the connections.
```

Comment:
This is so helpful! Thank you for making clear options here!  Thank you!! 

Can information to the README about mesh resolution and then another section on how to shift the mesh?  It will be extremely helpful to have all of the relevant information all together in the readme with the code.

Reply:
Thank you for the README updates @AliS-Noaa ! They are very helpful!